app/views/govuk_publishing_components/components/docs/breadcrumbs.yml

```diff
@@ -3,6 +3,8 @@ description: "Navigational breadcrumbs, showing page hierarchy"
 body: |
   Accepts an array of breadcrumb objects. Each crumb must have a title and a URL.
   Links have tracking data but links to the homepage (any link with a url of "/") will be tracked separately as `homeLinkClicked`
+
+  Note: Only the first and last items in the breadcrumb will be displayed on a page width less than 640px
```

Comment:
What's the plan with making the change so the last item isn't the current page? I assume it'll need deploying at the same time but the PR doesn't mention it.

Reply:
Am I right in thinking this has already been deployed?